registries/registry.go

```diff
@@ -217,6 +217,8 @@ func NewRegistry(configuration Config, asbNamespace string) (Registry, error) {
 		adapter = &adapters.LocalOpenShiftAdapter{Config: c}
 	case "helm":
 		adapter = &adapters.HelmAdapter{Config: c}
+	case "quay":
```

Comment:
Is there any reason we can't leverage the Jfrog work and make this one adapter? The code feels almost identical other than a few variations and both support v2 schema of Docker catalog API.

Reply:
I was thinking the same. It looks like this is using a quay-specific API, but if it's possible to just use the standard `/v2/_catalog` API, this can be combined with one or more other adapters.